### PR TITLE
add single template example to docs and pycbc test suite

### PR DESCRIPTION
**Pull request by ahnitz**

### CODE REVIEW

File: docs/inference/examples/single.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+This example demonstrates using the ``single_template`` model with the
+``emcee_pt`` sampler. First, we create the following configuration file:
+
+.. literalinclude:: ../../../examples/inference/single/single.ini
+   :language: ini
+
+:download:`Download <../../../examples/inference/single/single.ini>`
+
+For this example, we'll need to download gravitation-wave data for GW170817:
```
Comment (cdcapano) [inline docs/inference/examples/single.rst]: gravitation-wave -> gravitational-wave

File: docs/inference/examples/single.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   :language: bash
+   
+.. program-output:: bash ../examples/inference/single/run.sh
+
+:download:`Download <../../../examples/inference/single/run.sh>`
+
+This will run the ``emcee_pt`` sampler on the 2D analytic normal distribution with
+5000 walkers for 100 iterations. When it is done, you will have a file called
+``normal2d.hdf`` which contains the results. It should take about a minute to
+run.
```
Comment (cdcapano) [inline docs/inference/examples/single.rst]: I'm guessing you forgot to modify this paragraph from when you copied it from the doc for the analytic distribution.

**Comment by ahnitz** [comment]
@cdcapano LOSC went down, so I can't rebuild on ATLAS atm (good timing to go down between me building on my laptop and then trying on the cluster). I'll post what it looks like when I can build it on ATLAS.

**Comment by ahnitz** [comment]
@cdcapano GWOSC if finally back up. Here's what it looks like . https://www.atlas.aei.uni-hannover.de/work/ahnitz/projects/docs/infup2/latest/html/inference/examples/single.html

**Pull request merged**